src/Solver.cpp

```diff
@@ -624,11 +625,14 @@ void Solver::countFissionableFSRs() {
   log_printf(INFO, "Counting fissionable FSRs...");
 
   /* Count the number of fissionable FSRs */
-  _num_fissionable_FSRs = 0;
+  int num_fissionable_FSRs;
+#pragma omp parallel for reduction(+:num_fissionable_FSRs)
```

Comment:
Each thread gets is own copy of the local variable `num_fissionable_FSRs` (notice I had to create a local variable; OpenMP cannot do a reduction over a private variable) and each thread accumulates its own counter of fissionable FSRs. After leaving the for loop, the accumulators for each thread are added together.